app/code/core/Mage/Core/Model/Cache.php

```diff
@@ -664,6 +670,12 @@ public function cleanType($typeCode)
         $tags = $this->getTagsByType($typeCode);
         $this->clean($tags);
 
+        $path = self::XML_PATH_TYPES . '/' . $typeCode . '/tags';
+        $tagsConfig = Mage::getConfig()->getNode($path);
```

Comment:
question, instead of saving it in the config, could it be saved in the cache itself? if the cache gets reset externally this data would be out of sync. but in that case, how to get the date of the oldest entry in the cache? no idea